orchagent/portsorch.h

```diff
@@ -91,11 +92,13 @@ class PortsOrch : public Orch, public Subject
     void decreasePortRefCount(const string &alias);
     bool getPortByBridgePortId(sai_object_id_t bridge_port_id, Port &port);
     void setPort(string alias, Port port);
+    void erasePort(string alias);
```

Comment:
I dont think the function is used.. please revert to original file

Reply:
resolved in last commit